cli/utils/fs.ts

```diff
@@ -12,6 +13,12 @@ export function writeInCacheFolder(fileName: string, content: string, encoding:
     fs.writeFileSync(path.join(cachePath, fileName), content, { encoding });
 }
 
+export function writeToFile(fullPath: string, content: string, encoding: BufferEncoding = "utf8"): void {
+    const dir = path.dirname(fullPath);
+    mkdirp.sync(dir);
```

Comment:
Node already include a function to create folder recursively.
https://nodejs.org/api/fs.html#fsrmdirsyncpath-options

Reply:
removed deps. thanks!